app/src/pages/react-uswds-hello.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+  Alert,
+  Button,
+  Grid,
+  GridContainer,
+  Icon,
+} from "@trussworks/react-uswds";
+import Link from "next/link";
+import React from "react";
+
+/** Example page showing use of react-uswds components. Can be deleted once you're on a roll. */
+const HelloWorld = () => {
+  return (
+    <GridContainer>
+      <Grid row gap={5}>
+        <Alert type="success" headingLevel="h1" heading="Hello world">
+          This is an alert
+        </Alert>
+      </Grid>
+      <Grid row>
```

Comment:
❓ I feel like as a starting point if I were using this grid system I'd want this middle block to take up the most space. I also wasn't sure how to best implement that but I feel like showing how layout works would be helpful for folks. 

like if I was doing this with flex or similar I'd have the rows fill like .2 and .2 on bottom and the top and the middle auto fill the rest or something like that.

Reply:
One other thing I was wondering is if you can add margin around the rows (top or bottom margin specifically). 

I tried both offset and gap but had no luck 🤔

Reply:
> if you can add margin around the rows (top or bottom margin specifically).

I think the margin utility class should work

```tsx
<Grid row className="margin-bottom-3">
```

> I feel like as a starting point if I were using this grid system I'd want this middle block to take up the most space. I also wasn't sure how to best implement that but I feel like showing how layout works would be helpful for folks.

I dunno if there's an out-of-the-box way of doing this, without stringing together a number of utility classes, but I could be wrong. In a real app, I don't think this would happen at the `Grid` component level, since you'd have to factor in the overall viewport height (or other app-specific layout considerations). 

Untested pseudo code:

```tsx
<div className="minh-viewport display-flex flex-column">
  <Header>
    ...
  </Header>

  <main className="flex-fill">
    <GridContainer>...</GridContainer>
  </main>

  <Footer>...</Footer>
</div>
```